modules/training_pipeline/training_pipeline/api/inference.py

```diff
@@ -77,8 +77,8 @@ def load_data(self) -> Dataset:
         else:
             max_samples = 20
 
-        dataset = finqa.FinQADataset(
-            data_path=self._root_dataset_dir / "private_test.json",
+        dataset = qa.FinanceDataset(
+            data_path=self._root_dataset_dir / "training_data.json",
             scope=constants.Scope.INFERENCE,
```

Comment:
What about setting the loading file as a parameter and putting it as default to `testing_data.json`?  It is not fresh data, but at least we run the inference on the test data, not the training data.

Reply:
good point!
Will do ;)